src/components/shared/Slider/index.tsx

```diff
@@ -13,21 +16,46 @@ interface Props {
   step?: number;
 }
 
-const Slider: FC<Props> = () => {
+const Slider: FC<Props> = ({ maxValue = 100, minValue = 0 }) => {
+  const sliderRef = useRef<any>();
+  const [sliderWidth, setSliderWidth] = useState(0);
+  const [layoutX, setLayoutX] = useState(0);
+  const [positionX, setPositionX] = useState(0);
+
+  const getPercentagePosition = (): number => {
```

Comment:
What is `percentagePosition` for? :?

Reply:
I agree the variable name is weird. I will explain what it is below.